docs/plugins/slack.md

```diff
@@ -64,7 +64,7 @@ message to a specific user use `@handle` syntax:
     text: "Hi there!"
 ```
 
-> Though using a `@handle` does work, it will stop working if the user changes the _Display Name_
+> Though using `@handle` does work, it stops working, if the user changes the _Display Name_
```

Comment:
Hey, @mosabua is this better?

Reply:
lgtm

Reply:
Hey Manfred, could you help me merge this if it’s good?  I don’t have the proper access.

Reply:
I dont have access anymore either...